src/components/manager/runs/RunLastContent.js

```diff
@@ -0,0 +1,102 @@
+import React, {useState, useEffect} from "react";
+import {Table} from "antd";
+import {connect} from "react-redux";
+import PropTypes from "prop-types";
+import {fetchRuns} from "../../../modules/wes/actions";
+
+const LastIngestionTable = ({runs, fetchRuns}) => {
+    const [ingestions, setIngestions] = useState([]);
+
+    const processIngestions = (data) => {
+        const latestIngestions = data.reduce((acc, obj) => {
+            if (obj.state === "COMPLETE") {
+                const dataType = obj.details.request.tags.workflow_metadata.data_type;
+                const tableId = obj.details.request.tags.table_id;
+                const fileNameKey = Object.keys(obj.details.request.workflow_params)[0];
+                const filePath = obj.details.request.workflow_params[fileNameKey];
+                const fileName = filePath.split("/").pop();
+                const date = obj.details.run_log.end_time.split("T")[0];
+
+                const currentIngestion = { date, dataType, tableId, ingestedFiles: 1, fileNames: [fileName] };
+
+                const existingIngestion = acc.find((elem) => elem.dataType === dataType);
+
+                if (!existingIngestion) {
+                    acc.push(currentIngestion);
+                } else if (date > existingIngestion.date) {
+                    Object.assign(existingIngestion, currentIngestion);
+                } else if (date === existingIngestion.date) {
+                    existingIngestion.ingestedFiles += 1;
+                    existingIngestion.fileNames.push(fileName);
+                }
+            }
+            return acc;
+        }, []);
+
+        return latestIngestions;
+    };
+
+    useEffect(() => {
+        async function fetchIngestions() {
+            try {
+                await fetchRuns({with_details: true});
```

Comment:
is this needed? i thought this page already auto-fetched runs

Reply:
No needed, removed!